web/src/stream_data.ts

```diff
@@ -365,29 +365,19 @@ export function get_streams_for_user(user_id: number): {
     };
 }
 
-export function get_invite_stream_data(): InviteStreamData[] {
-    function get_data(sub: StreamSubscription): InviteStreamData {
-        return {
-            name: sub.name,
-            stream_id: sub.stream_id,
-            invite_only: sub.invite_only,
-            is_web_public: sub.is_web_public,
-            default_stream: default_stream_ids.has(sub.stream_id),
-        };
-    }
-
+export function get_invite_stream_data(): StreamSubscription[] {
```

Comment:
Should this function be modified to also include all the public streams?

Reply:
I think you should try posing the question as to what needs to be decided in #frontend, with appropriate context. Probably someone will be able to help.

Reply:
Here is the [CZO ](https://chat.zulip.org/#narrow/stream/6-frontend/topic/invite.20streams/near/1824320) chat.